modules/main.tf

```diff
@@ -1,104 +1,150 @@
-resource "spacelift_module" "sage-aws-vpc" {
-  github_enterprise {
-    namespace = "Sage-Bionetworks-Workflows"
-    id        = "sage-bionetworks-workflows-gh"
-  }
+locals {
+  spacelift_modules = {
 
-  name               = "sage-aws-vpc"
-  terraform_provider = "aws"
-  administrative     = false
-  branch             = "main"
-  description        = "Terraform module for creating a VPC in AWS"
-  repository         = "eks-stack"
-  project_root       = "modules/sage-aws-vpc"
-  space_id           = "root"
-}
+    eks = {
+      github_enterprise = {
+        namespace = "Sage-Bionetworks-Workflows"
+        id        = "sage-bionetworks-workflows-gh"
+      }
+      repository = "eks-stack"
 
-resource "spacelift_version" "sage-aws-vpc-version" {
-  module_id      = spacelift_module.sage-aws-vpc.id
-  version_number = "0.3.3"
-}
+      name               = "sage-aws-eks"
+      terraform_provider = "aws"
+      administrative     = false
+      branch             = var.git_branch
```

Comment:
Can something like this not be used in the other places where we pass a `branch` now?

Reply:
I added these changes to https://github.com/Sage-Bionetworks-Workflows/eks-stack/pull/15